Kontent.Ai.Management.Tests/ManagementClientTests/WebSpotlightTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Kontent.Ai.Management.Models.Shared;
+using Kontent.Ai.Management.Models.WebSpotlight;
+using Kontent.Ai.Management.Tests.Base;
+using System;
+using System.Net.Http;
+using Xunit;
+using static Kontent.Ai.Management.Tests.Base.Scenario;
+
+namespace Kontent.Ai.Management.Tests.ManagementClientTests;
+
+public class WebSpotlightTests : IClassFixture<FileSystemFixture>
+{
+    private static readonly string WebSpotlightBaseUrl = $"{Endpoint}/projects/{ENVIRONMENT_ID}/web-spotlight";
+    private readonly Scenario _scenario = new(folder: "WebSpotlight");
+
+    [Fact]
+    public async void ActivateWebSpotlight_Returns_EnabledStatusAndRootTypeId()
```

Comment:
nitpick: I checked other tests and we are not 100% consistent on signatures for xunit tests. some use `async Task`, others `async void`. if I understand, xunit has custom context which allows test method signatures with `async void`, so this is not really a problem, rather my observation :)